src/poprox_recommender/handler.py

```diff
@@ -30,7 +30,13 @@ def generate_recs(event, context):
     else:
         logger.info("Using default parameters")
 
-    logger.info("Selecting articles...")
+    num_candidates = len(req.todays_articles)
```

Comment:
`if num_candidates < req.num_recs` ?

Reply:
Yeah, I guess that prevents the two article newsletter. Will update